sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/NumberConverter.scala

```diff
@@ -89,6 +91,10 @@ object NumberConverter {
     var i = fromPos
     while (i < value.length) {
       value(i) = Character.digit(value(i), radix).asInstanceOf[Byte]
+      // if invalid characters are found, it no need to convert the suffix starting there
```

Comment:
can we update the method doc as well? it currently says `Convert invalid characters to -1.`

Reply:
Updated! Thanks